### PR TITLE
#166901539 Improve application's documentation

### PR DESCRIPTION
**Pull request by aayebare**
#### What Does This PR Do?
- Improves application's documentation
#### Description Of Task To Be Completed
- Remove grammatical errors
- Add more context to project set up documentation
#### Any Background Context You Want To Provide?
- The current documentation setup has proven to be confusing for developers who set up the application for the first time on the local development machines.

#### What are the relevant pivotal tracker stories?
[#166901539]
#### Screenshot(s)
None

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
           command: |
             . venv/bin/activate
             export DJANGO_SETTINGS_MODULE=a_socials.settings.dev
-            python server/manage.py test --snapshot-update server/api/tests/ server/graphql_schemas/tests/
+            python server/manage.py test --snapshot-update server/api/tests/ server/graphql_schemas/tests/ --keepdb
```
Comment (brianbirir) [inline .circleci/config.yml]: This file should not be part of this PR. Please remove it @m-asiimwe

File: client/__tests__/components/cards/EventCard.test.js [hunk 2]
```diff
@@ -7,6 +7,6 @@ import EventCard from '../../../components/cards/EventCard';
 describe('Search Modal', () => {
```
Comment (brianbirir) [inline client/__tests__/components/cards/EventCard.test.js]: This file should not be part of this PR. The focus was on `README.md`

**Review by brianbirir — CHANGES_REQUESTED** [review]
The `README.md` file should be the only one part of this review. Please remove the other two.

**Comment by brianbirir** [comment]
@m-asiimwe thanks for this. Please see my comments.

File: .circleci/config.yml (hunk 1 above)
Comment (aayebare) [reply .circleci/config.yml]: noted. Though without this change, the circleci build is bound to fail

File: README.md [hunk 3]
```diff
@@ -1,46 +1,49 @@
...
+With this platform, events organized within the fellowship are properly tracked and managed for all to see and for all to relate with irrespective of their centers.
+This will improve socialization among Andelans both inside and outside the organisation.
 
-Andela Social was born out of the desire to keep Andelans very sociable within the organization.
-It is a platform for events aggregation, to get Andelans from different departments to mingle and have fun together over a cup of coffee, a game of soccer, at a swimming pool or at a friends dinner party.
-It is a platform to get events organized within the fellowship properly tracked and managed for all to see and for all to relate with irrespective of their centers.
-These will improve socialization between Andelans inside and outside the company.
+## PROJECT REQUIREMENTS
+- Node
+- Postgres Database
```
Comment (brianbirir) [inline README.md]: Please add the correct spelling and version number e.g. `PostgreSQL v. 10.9`

**Review by brianbirir — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (brianbirir) [reply README.md]: This is ambiguous. Node has different meanings. Please specify the version and correct terminology i.e. `Node.js 8.x.x`

Comment (brianbirir) [reply README.md]: There are other project requirements such as Python, virtualenv etc.
